main.tf

```diff
@@ -616,6 +616,34 @@ module "aws_for_fluentbit" {
     {
       name  = "serviceAccount.name"
       value = local.aws_for_fluentbit_service_account
+    },
+    {
+      name  = "cloudWatch.region"
+      value = local.region
+    },
+    {
+      name  = "cloudWatchLogs.region"
+      value = local.region
+    },
+    {
```

Comment:
I would be open to setting CloudWatch values here, but the others are valid for cross-region log shipping and should be left to users

Reply:
done